src/components/SubscribeButton/NotificationModal.tsx

```diff
@@ -198,11 +198,11 @@ export default function NotificationModal({
                 </ButtonTextt>
               </ButtonOutlined>
             </ActionWrapper>
-          </>
+          </div>
         )
       case VIEW.SUBSCRIBE_SUCCESS:
         return (
-          <>
+          <div>
```

Comment:
why do you add a redundant div here? what happen with fragment

Reply:
I don't need gap effect by it's parent.

Reply:
then, just remove gap from `ContentWrapper` https://github.com/KyberNetwork/kyberswap-interface/pull/1477/files#diff-bb86fa0ce5436db6d998b2222602c8b4ea74c570b8d2bd3bca2bf91d4c5bf3acR28

Reply:
<Flex gap=...>
.... 
I need gap here
<div>
   my component here, I don't need gap here
</div>
</Flex>

Reply:
okay, got it

Reply:
thank you